### ISSUE TITLE
Plugin don't create android-sources, javadoc and pom-default

### ISSUE THREAD

**Issue by StefMa** [StefMa/AndroidArtifacts#8]
Sometimes, I don't know exactly in which cases, the following files won't be created:
* myLibName-sources.jar (located in `myLibName/build/libs/`)
* myLibName-javadoc.jar (located in `myLibName/build/libs/`)
* pom-default.xml (located in `myLibName/build/publications/release/`)

To manually created it we can run the following task (same order like above):
* [release]AndroidSourcesJar
* [release]AndroidJavadocsJar
* generatePomFileFor[Release]Publication

The final (and maybe always working) gradle-wrapper command would be:
```
./gradlew clean build releaseAndroidSourcesJar releaseAndroidJavadocsJar generatePomFileForReleasePublication artifactoryPublish
```

Think how to proceed here.
Maybe add dependencies to `artifactoryPublish` tha it should run after the other tasks...

**Comment by passsy** [comment]
This could be fixed with #9

**Comment by StefMa** [comment]
Definitely fixed with #14
